tools/ratbagctl.in

```diff
@@ -719,7 +719,7 @@ parser_def = [
                     func: func_led_get,
                 },
                 {
-                    of_type: set,
+                    of_type: switch,
```

Comment:
The commit I am talking about: https://github.com/bentiss/libratbag/commit/ff4f0205392463afd41c67d628486e809c77227f